java-checks/src/main/java/org/sonar/java/checks/AssertionsInTestsCheck.java

```diff
@@ -115,7 +116,10 @@
     method("com.github.tomakehurst.wiremock.WireMockServer", VERIFY).withAnyParameters(),
     // Eclipse Vert.x
     method("io.vertx.ext.unit.TestContext", STARTS_WITH_ASSERT).withAnyParameters(),
-    method("io.vertx.ext.unit.TestContext", STARTS_WITH_FAIL).withAnyParameters());
+    method("io.vertx.ext.unit.TestContext", STARTS_WITH_FAIL).withAnyParameters(),
+    // Selenide
+    method( "com.codeborne.selenide.SelenideElement", NameCriteria.startsWith(SHOULD) ).withAnyParameters(),
+    method( "com.codeborne.selenide.ElementsCollection", NameCriteria.startsWith(SHOULD) ).withAnyParameters());
```

Comment:
This is minor, but to be consistent, our [code style](https://github.com/SonarSource/sonar-developer-toolset#code-style) does not allow spaces after/before opening/closing parentheses.